### DISCUSSION TITLE
Criei uma ordem de estudo, quero opiniões de pessoas mais experientes!

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#2460] [Perguntas e Respostas]
Oi gente! Fiz um planejamento de estudo pra aprender webdev, pelo menos começar os estudos, dai eu fiz esse plano de estudo meio lógico pra tentar entender. O que vocês acham? 

```
1. Khan Academy.
1.1 Cursos de matemática (só pra ter uma base melhor, faz quase 11 anos que sai do ensino médio e tá complicado, n lembro de quase nada)

2. CS50 Harvard CC Course.

? OSSU - Open Source Society University (Aqui seria uma boa ideia?)

3. Curso em Vídeo:
3.1 Algoritmo; 
3.2 HTML & CSS; 
3.3 JS puro;

4 Git e GitHub

5. Framework
5.1 React
5.2 Vue.js
5.3 Angular

```
~enquanto isso tenho lido: _Learning Web Design: A BEGINNER'S GUIDE TO HTML, CSS, JAVASCRIPT, AND WEB GRAPHICS_ no tempo livre no trabalho.

Por enquanto só cheguei até ai, se tiverem sugestões a adicionar eu ficarei muito grato!  Eu sei que tem uns frameworks e coisas além, então eu vou por partes.

Estou seguindo também esse [roadmap.sh](https://roadmap.sh/frontend) mas gosto de ter algo mais personalizado pra mim. 

Edit: Também estou pensando em seguir o currículo do [OSSU - Open Source Society University](https://github.com/ossu/computer-science), mas não saberia em que ordem colocar ele, alguém tem sugestões? Talvez depois do CS50

**Comment by nazarepiedady** [comment]
@kaiomazziero, uma recomendação que dou-te é a de não ler muito, não ver muitos vídeos, e principalmente não variar muito no conteúdo.

Repita sempre que possível o mesmo conteúdo até os assuntos tornarem-se naturais, dependendo da tua dedicação isso pode durar uns 2 à 3 meses.

A ordem do material que tencionas estudar está relativamente boa, no entanto recomendo na secção de abstrações (**frameworks**) trocar a ordem para:

- Vue ou Svelte (a estrutura destas abstrações são muito próximas das que usamos quando escrevemos HTML, CSS, e JavaScript puros, o que naturaliza a transição)

- React ou Angular (a estrutura destas duas é o contrário das que destaquei acima)

**Comment by maxxcleiton** [comment]
Também não recomendo viajar muito em leituras e vídeos.
Se tu quer desenvolver, eu procuraria atividades práticas para fazer, e praticar conceitos e lógicas
Entender linhas de raciocínio não só na teoria, mas na prática principalmente, daí batendo a cara na prática, correndo pra teoria e assim continuando.
